main.py

```diff
@@ -3,48 +3,56 @@
 from selenium import webdriver
 from selenium.webdriver.chrome.options import Options
 import discord
+from discord.ext import commands
+from discord_slash import SlashCommand
+from discord_slash.utils import manage_commands
 import datetime
 import json
 import re
 import html2text
+import sys
 
 def jsonFromFile(filePath):
   with open(filePath,"r",encoding="utf8") as file:
-    return json.loads(file.read())
+    return json.load(file)
 
 #top secret stuff !
-pws = jsonFromFile("pws.json")
-pw = pws["pw"]
-username = pws["username"]
-token = pws["token"]
+settings = jsonFromFile("settings.json")
+pw = settings["password"]
+username = settings["username"]
+token = settings["token"]
+#admin ids
+admins = settings["admins"]
+# server id (singular)
+guild_id = settings["guild_id"]
 
 #Stuff Stack Overflow told me to put to stop crashes ¯\_(.-.)_/¯
 browser_options = Options()
 browser_options.add_argument('--no-sandbox')
 browser_options.add_argument('--disable-dev-shm-usage')
 
 #headless !!!
-#browser_options.add_argument("headless")
+if "headless" in settings and settings["headless"]:
+  browser_options.add_argument("headless")
```

Comment:
```suggestion

```
Headless doesn't actually work in my experience so I would delete this if we're nuking it anyways.

Reply:
Resolved.